src/main/webapp/js/app.js

```diff
@@ -118,7 +120,100 @@ angular
             parent: 'root'
         });
 
-        }
+        $translateProvider.translations('en', {
```

Comment:
May be, it makes sense to move translations in separate files - 1 file for 1 language.